packages/cardano-graphql/src/Schema/types/Reward.ts

```diff
@@ -0,0 +1,26 @@
+import { Cardano } from '@cardano-sdk/core';
+import { Directive, Field, Int, ObjectType } from 'type-graphql';
+import { Epoch } from './Epoch';
+import { Int64 } from './util';
+import { RewardAccount } from './Address';
+import { StakePool } from './StakePool';
+
+@ObjectType()
+export class Reward {
+  @Field(() => RewardAccount)
+  rewardAccount: RewardAccount;
+  @Directive('@search')
+  @Field(() => Int)
+  epochNo: Cardano.Epoch;
+  @Field(() => Epoch)
+  epoch: Epoch;
+  @Field(() => Int)
+  spendableAtEpochNo: Cardano.Epoch;
+  @Field(() => Int64)
+  quantity: Cardano.Lovelace;
+  @Directive('@search(by: [exact])')
+  @Field(() => String, { description: 'poolMember | poolLeader | treasury | reserves' })
+  source: string;
+  @Field(() => StakePool, { description: "null when source is 'treasury' or 'reserves'", nullable: true })
```

Comment:
```suggestion
  @Field(() => StakePool, { description: "null when source is 'poolLeader', 'treasury', 'reserves'", nullable: true })
```

Reply:
Probably not null for 'poolLeader'?

Reply:
Ah yes you're right. I was thinking the field was only there to associate member rewards